File: 06.17/Exception/src/Ex.java

public class Ex {
	int[] arr = {1, 2, 3, 4, 5};
	
	public void printAllElement() {
		for(int i = 0; i < arr.length; i++)
			System.out.println(arr[i]);
	}
	
	public void printOneElement(int num) {
		// 예외처리하고 정상 범위 알려주기
		try {
			System.out.println(arr[num - 1]);
		} catch (ArrayIndexOutOfBoundsException e) {
			System.out.println("1~" + arr.length + " 사이의 정수를 입력하세요");
		}
	}
}

File: 07.08/workspace_web/ex.js
function digits(num){
    let i = 1;
    let inputNum = num;
    for(;;){
        if(inputNum/10 >= 1){
            i++;
            inputNum /= 10;
        } else
            break;
    }

    return i;
}

function digits2(num){
    let str = '' + num;
    return str.length;
}


function printSquared(arr){
    
    for(let num of arr){
        console.log(num * num);
    }
    // in을 쓰면 인덱스(key)를 반환
    // of를 쓰면 값(value)를 반환
}